test/support/protocol_test_case.rb

```diff
@@ -155,20 +155,6 @@ def req_terminate_debuggee
       close_reader
     end
 
-    def assert_reattach
```

Comment:
If you want to use `req_disconnect` and `attach_to_cdp_server`, please move them into API block and update document.

https://github.com/ruby/debug/blob/master/test/support/protocol_test_case.rb#L41

Reply:
👍 done